scania-digital-design-system/sdds
# Improvement - gulp file improvement

**Describe pull-request**  
_Describe what the pull-request is about_
Added a `try and catch` for the theme building process, making it easier to see if an error occurred. 
Also fixed issue with auto-generated files being included in the prettier process, making it contain a change every time we built. Also ignoring `.angular` folder containing auto-generated files for prettier.

**How to test**  
_Add description how to test if possible_
1. Go to theme/light folder
2. Add a change that causes an error in the build process, in any scss file
3. Run the build process a more verbose message will appear for more details
4. Revert back to original state without the error, run the build so it works

---
1. run the prettier, it should skip the auto-generated files `.angular` and in the core `theme/core/spacing/spacing-vars.scss` and `theme/core/typography/typography-vars.scss `
